### ISSUE TITLE
Amazon MTurk

### ISSUE THREAD

**Issue by simonMadec** [kyamagu/js-segment-annotator#34]
Thanks for this great implementation!
Is it possible to use it in Amazon MTurk?

**Comment by kyamagu** [comment]
@simonMadec Yes, it should be possible. Everything is implemented in Javascript. Just need to make sure workers understand the task.

**Comment by mia2mia** [comment]
@kyamagu Or do you know any self contained javascript code for image segmentation which i can directly use on mturk. I can use labelme for license issues. I currently use vgg but dont know how to port it to aws
